### ISSUE TITLE
MSYS2 Could not resolve host

### ISSUE THREAD

**Issue by AndreaPisoni** [Alexpux/MSYS2-packages#1455]
Just installed MSYS2, I cannot install any package, it seems the DNS is not working, however, it works fine with any other software installed on my machine. 

Example output:

```
$ pacman -Syuu
:: Synchronizing package databases...
error: failed retrieving file 'mingw32.db' from repo.msys2.org : Could not resolve host: repo.msys2.org
```

What can it be?

**Comment by elieux** [comment]
Can you resolve any other host in MSYS2?

-- 
David Macek

**Comment by AndreaPisoni** [comment]
No, I cannot resolve any host in MSYS2. I tried to use wget and curl as well to reach simple domains but it fails. 

Other software like conda and pip can resolve domains and install packages without issues, maybe MSYS2 has its own DNS config?

**Comment by AndreaPisoni** [comment]
Hosts cannot be resolved during installation either:

```
==> Updating trust database...

gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
```

I am behind a proxy, but my env variables HTTP_PROXY and HTTPS_PROXY are set up and work with all my existing software.

**Comment by mcjh** [comment]
I also have this problem after installing msys2 on win10.  Msys2 on win7 works fine.

**Comment by mcjh** [comment]
Turned out to be because of (I believe) server fire-wall settings/controls.  Problem went away when I used a different network.

**Comment by Priyatham10** [comment]
When you're behind a proxy open ./msys64/

> Hosts cannot be resolved during installation either:
> 
> ```
> ==> Updating trust database...
> 
> gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
> gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
> gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
> gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
> gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
> gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
> gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
> gpgkeys: HTTP fetch error 6: Could not resolve host: pool.sks-keyservers.net
> ```
> I am behind a proxy, but my env variables HTTP_PROXY and HTTPS_PROXY are set up and work with all my existing software.

When you're behind proxy, instead of setting proxy using environment variables, try opening ./msys64/network.xml and edit the "Host and Port" for your network as shown below. Edit the "Username and Password" too if you like.

![InkedCapture_LI](https://user-images.githubusercontent.com/17174106/54865416-79287c00-4d8b-11e9-8287-6347ca091adc.jpg)

**Comment by tardyp** [comment]
It looks like pacman wants https_proxy and http_proxy, but not HTTPS_PROXY or HTTP_PROXY.

The funny thing on Windows is that environment variables are case insensitive. 
So if you set *both*, then windows will actually only store the last one.

Some other programs only look at the uppercase version, so we are screwed if we want to make a sensible system configuration.
